.husky/pre-commit

```diff
@@ -4,4 +4,4 @@
 export NVM_DIR="$HOME/.nvm"
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
 
-npx lint-staged
+cat .lintstagedrc.json | npx lint-staged --config -
```

Comment:
How come is this required? Isn't lint-staged picking up the config implicitly when using `npx lint-staged`?